components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -107,7 +138,7 @@ export default function RepositoryFinder(props: RepositoryFinderProps) {
                 <div className="truncate w-80">{displayContextUrl(props.initialValue) || "Select a repository"}</div>
```

Comment:
**thought**: When users select a repository that's already in the list as a repository or project, we should probably surface the selection in a similar structure, not just the plain repository URL. For example, we already allow users to rename projects, but this should be more important as we allow multiple instances of the same repository, etc.

<img width="499" alt="Screenshot 2023-09-14 at 20 16 39" src="https://github.com/gitpod-io/gitpod/assets/120486/c989391b-9351-4585-b92a-210a6aff3234">

Reply:
Ah yah, good catch.